ml_metadata/metadata_store/mysql_metadata_source.cc

```diff
@@ -120,6 +120,10 @@ Status MySqlMetadataSource::ConnectImpl() {
                   ssl.ca().empty() ? nullptr : ssl.ca().c_str(),
                   ssl.capath().empty() ? nullptr : ssl.capath().c_str(),
                   ssl.cipher().empty() ? nullptr : ssl.cipher().c_str());
+    if (ssl.has_verify_server_cert() && ssl.verify_server_cert()) {
```

Comment:
note `verify_server_cert` is a primitive type proto field, instead of the if ( ... ) { } , do the following: 

```
my_bool verify_server_cert = ssl.verify_server_cert() ? 1 : 0;
mysql_options(db_, MYSQL_OPT_SSL_VERIFY_SERVER_CERT, &verify_server_cert);
```